CinC2016Challenge_PCG/examples/nupic2016/wav2vect.m

```diff
@@ -0,0 +1,48 @@
+%% Load the trained parameter matrices for Springer's HSMM model.
+% The parameters were trained using 409 heart sounds from MIT heart
+% sound database, i.e., recordings a0001-a0409.
+load('Springer_B_matrix.mat');
+load('Springer_pi_vector.mat');
+load('Springer_total_obs_distribution.mat');
+springer_options   = default_Springer_HSMM_options;
+springer_options.audio_Fs = 1000 %TODO subsample to which value? (higher=longer learning, lower=worse data)
+
+%% Load data and resample data
+normals = importfile('normals.csv');
+N=100 %FIXME size(normals)
+prob=0.01; % to make this faster, use only random 2\% of the files; %FIXME process all files
+result=[]; % select signal/features used for training
+signal=[]; % whole PCG signal, unprocessed
+for i=1:N
+    r=normals{i};
+    r=r(2:end-1);
+    [PCG, Fs1] = audioread([r '.wav']);  % load data
```

Comment:
I'd like to create several audio encoders (#22 ), and be able to use this project as `nupic.vision`, installable through pip. 
* one of the implementations could be based on matlab/octave!